src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -151,6 +169,39 @@ public void parse_configFolderAndOutputDirectory_success() throws ParseException
         Assert.assertTrue(Files.isSameFile(expectedAbsoluteOutputDirectoryPath, cliArguments.getOutputFilePath()));
     }
 
+    @Test
+    public void parse_configFolderandViewWithouthArgs_returnsConfigCliArguments() throws ParseException, IOException {
+        String input = String.format("-config %s -view", CONFIG_FOLDER_ABSOLUTE);
+        CliArguments cliArguments = ArgsParser.parse(translateCommandline(input));
+
+        Assert.assertTrue(cliArguments instanceof ConfigCliArguments);
+        Assert.assertTrue(Files.isSameFile(
+                REPO_CONFIG_CSV_FILE, ((ConfigCliArguments) cliArguments).getRepoConfigFilePath()));
+        Assert.assertTrue(Files.isSameFile(
+                AUTHOR_CONFIG_CSV_FILE, ((ConfigCliArguments) cliArguments).getAuthorConfigFilePath()));
+        Assert.assertTrue(cliArguments.isAutomaticallyLaunching());
+
+        input = String.format("-config %s -view", CONFIG_FOLDER_RELATIVE);
+        cliArguments = ArgsParser.parse(translateCommandline(input));
+
+        Assert.assertTrue(cliArguments instanceof ConfigCliArguments);
+        Assert.assertTrue(Files.isSameFile(
+                REPO_CONFIG_CSV_FILE, ((ConfigCliArguments) cliArguments).getRepoConfigFilePath()));
+        Assert.assertTrue(Files.isSameFile(
+                AUTHOR_CONFIG_CSV_FILE, ((ConfigCliArguments) cliArguments).getAuthorConfigFilePath()));
+        Assert.assertTrue(cliArguments.isAutomaticallyLaunching());
+    }
+
+    @Test
+    public void parse_configFolderAndViewWithArgs_returnsViewCliArguments() throws ParseException, IOException {
+        String input = String.format("-config %s -view %s", CONFIG_FOLDER_ABSOLUTE, OUTPUT_DIRECTORY_ABSOLUTE);
```

Comment:
Hmm, this shouldn't be valid?

Line 197 implies that such command is allowed -> 
`java -jar RepoSense.jar -config path/to/configs -output . -view /etc/secrets`
Note: When `-output` is not provided, it will use `cwd` **.**

As -view is atomic, we can merge commands `1.` and `2.` to `3.`
1. `java -jar RepoSense.jar -config path/to/configs -output path/to/output`
2. `java -jar RepoSense.jar -view path/to/output`
3. `java -jar RepoSense.jar -config path/to/configs -output path/to/output -view`

Since  `1.` and `2.` are equivalent to `3.`, output and view should use the same `PATH`. 
It seems weird for `-view` to accept another `PATH`.

Reply:
The main issue is `ViewCliArguments` is returned. So, there will not be any code analysis.

Reply:
There is some issue with this because there is no easy way to differentiate these 2 commands:
1. `java -jar RepoSense.jar -config path/to/configs -view /etc/secrets`
2. `java -jar RepoSense.jar -view /etc/secrets`

This is because `-config` parameter has a default value which is the cwd. When we parse the command line arguments for both commands, the values for `-config` and `-view` are non-null. So it might be hard to detect when a command like the one below is given.
- `java -jar RepoSense.jar -config path/to/configs -output . -view /etc/secrets`

Reply:
@myteo 

Sorry I didn't follow the previous PR that closely, it seems that it was agreed to keep it like this. So, disregard this comment for now. 👍